src/web_app_skeleton/src/actions/api_action.cr

```diff
@@ -1,3 +0,0 @@
-abstract class ApiAction < Lucky::Action
-  # Add pipes and methods that are for all API requests
-end
```

Comment:
I think this can be left in. Full web apps should also have an ApiAction

Reply:
I think this was renamed to the `api_action.cr.ecr` file below